src/WinRT.Runtime/Projections/INotifyCollectionChanged.net5.cs

```diff
@@ -35,21 +37,22 @@ public static unsafe (Action<global::System.Collections.Specialized.NotifyCollec
                     (*(delegate* unmanaged[Stdcall]<IntPtr, global::WinRT.EventRegistrationToken, int>**)ThisPtr)[7]);
             });
             return eventSource.EventActions;
-        }
-
-        public static global::System.Guid IID { get; } = new Guid(new global::System.ReadOnlySpan<byte>(new byte[] { 0xE1, 0x55, 0x01, 0x53, 0xA5, 0x28, 0x93, 0x56, 0x87, 0xCE, 0x30, 0x72, 0x4D, 0x95, 0xA0, 0x6D }));
-
-        public static IntPtr AbiToProjectionVftablePtr => INotifyCollectionChanged.Vftbl.AbiToProjectionVftablePtr;
+        }
+
+        public static global::System.Guid IID { get; } = GuidGenerator.GetWuxMuxIID(typeof(INotifyCollectionChanged).GetCustomAttribute<WuxMuxProjectedTypeAttribute>());
```

Comment:
Can we not hardcode this by switching on the flag here too, to avoid the attribute lookup?